### ISSUE TITLE
Advanced Search/Related Resources Panels - 3.3.2 Labels or Instructions - 1 missing form label is present on the search field

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9856]
As a user reliant on screen reader to interpret page components, I expect all form fields in the page to have accessible labels which provide clear, succinct descriptions of the fields so I am aware of what the field does and how to use it.

**Linked work items:**
#9386 #9388 

**Expected Result:**
All form fields should have accessible labels which give a clear, succinct description of the name/purpose/instructions for use of the form field.

**Actual Result:**
 1 missing form label is present on the search field. Missing form labels will cause incorrect screen reader interpretation of form fields. The screen reader simply reads, 'Find, edit blank', rather than the name of the field.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/be58fa67-1d5c-4776-9f6e-46969861f5ba)

![Image](https://github.com/archesproject/arches/assets/77161085/eda27d8a-dcc6-43c6-b850-388528b5da6e)

**Comment by johnatawnclementawn** [comment]
This will be resolved by #9829

**Comment by apeters** [comment]
@imogenroberts can you retest, this should be fixed in the latest

**Comment by imogenroberts** [comment]
RETEST: WAVE no longer flags the search bar as having a missing form label. The search bar has an accessible and instructive label, (e.g. 'Filter the related resources network')

**Evidence:**
![Related Resources - 3 3 2 Labels or Instructions FIXED](https://github.com/archesproject/arches/assets/77161085/8a566547-d449-4e39-be17-0a277dc3badf)
